package.json

```diff
@@ -9,23 +9,23 @@
   "author": "james.simone",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "latest",
-    "@babel/eslint-parser": "latest",
-    "@jongpie/sfdx-bummer-plugin": "latest",
-    "@ljharb/eslint-config": "latest",
-    "@lwc/eslint-plugin-lwc": "latest",
-    "@prettier/plugin-xml": "latest",
-    "@salesforce/eslint-config-lwc": "latest",
-    "@salesforce/eslint-plugin-lightning": "latest",
-    "@salesforce/sfdx-lwc-jest": "0.12.5",
-    "@salesforce/sfdx-scanner": "3.16.0",
-    "eslint": "latest",
-    "eslint-config-prettier": "latest",
-    "husky": "latest",
-    "lint-staged": "latest",
-    "prettier-plugin-apex": "latest",
-    "prettier": "latest",
-    "sfdx-cli": "latest"
+    "@babel/core": "^7.23.3",
```

Comment:
please pin dependencies in this file

Reply:
Remove the carat, or move all to "latest"?

Reply:
Remove the carat!

Reply:
Ah - a solid move. I thought "latest" was risky, so I reduced it to the range. Pinning is even better. Fully agree.